broker_test.go

```diff
@@ -41,6 +42,8 @@ func TestSubscribe(t *testing.T) {
 		testTopic := URI("turnpike.test.topic")
 		msg := &Subscribe{Request: 123, Topic: testTopic}
 		broker.Subscribe(sess, msg)
+		// broker.Subscribe() runs Send() in a separate go routine, so we need to give it time to complete
+		time.Sleep(100 * time.Millisecond)
```

Comment:
Note that instead of sleep you can also use Gomega's or Testify's `Eventually()` to make async assertions. It's perfect for cases where you need to wait for some data to show up in a query or on a channel.

Reply:
That's a good point. The tests were broken before I even started and I just wanted to fix them so I could move on, so I did the simplest thing I could think of... I think I have more work to do on this package and I'll clean this up when I next work on it.